UPGRADE-1.9.md

```diff
@@ -2,32 +2,12 @@
 
 ### Package upgrades
 
-1. Add new bundles to your list of used bundles in `config/bundles.php` if they are not already there:
-
-    ```diff
-    +   BabDev\PagerfantaBundle\BabDevPagerfantaBundle::class => ['all' => true],
-    +   SyliusLabs\Polyfill\Symfony\Security\Bundle\SyliusLabsPolyfillSymfonySecurityBundle::class => ['all' => true],
+1. Upgrade the version of `friendsofsymfony/oauth-server-bundle` by:
 
-1. Remove `getContainerLoader` method from `src/Kernel.php` class if you did not customise it.
-
-1. We've removed the support for Symfony's Templating component (which is removed in Symfony 5). 
+    ```bash
+    composer require "friendsofsymfony/oauth-server-bundle" ">2.0.0-alpha.0 ^2.0@dev"
```

Comment:
Is it required when staying with Symfony 4.4?

Reply:
It looks as it is because without this requirement I have that kind of error: https://github.com/GSadee/TestUpgrade/runs/1877377582?check_suite_focus=true#step:22:10

Reply:
Ah, that's right :/